Lib/gftools/builder/__init__.py

```diff
@@ -334,6 +334,13 @@ def run_fontmake(self, source, args):
         # https://github.com/googlefonts/fontmake/issues/872
         args["filters"] = [...] + filters
 
+        if source.endswith((".glyphs", ".designspace")):
+            args["expand_features_to_instances"] = self.config.get(
+                "expandFeaturesToInstances", True
+            )
+        args["reverse_direction"] = self.config.get("reverseOutlineDirection", True)
+        args["remove_overlaps"] = self.config.get("removeOutlineOverlaps", True)
+
```

Comment:
I'm assuming that by not setting these fields, we'll get fontmake defaults? if we don't, we'll end up breaking existing projects.

Reply:
Yas, removing overlaps and reversing direction are the default in fontmake. Expanding the features is off by default, but projects are more likely to break that way and turning it on is the only sensible option I think.

Reply:
If the fields are not in self.config, should we maybe not pass anything to fontmake, and let fontmake fill-in the default values? That way the default values are stored in only one place, fontmake.

Reply:
Done.